### PR TITLE
Update readme to clarify methods implementation

### PR DESCRIPTION
**Pull request by hegate**
Including a bigger paragraph to clarify current method implementation

# Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [ ] All new and existing tests passed.
- [ ] The code follows the code style and conventions of this project.
- [ ] The change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

# Description of the change

...

# Motivation for the change

...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Overview
 > Plus, a lot of great functionality will be coming in the next few 
 > months, including:
 >
-> * Extended method support for your simulated devices to help demo and test additional scenarios
+> * Extended method support for your simulated devices to help demo and test additional scenarios. Details about the Contoso sample devices and rules can be found in the documentation [here](https://docs.microsoft.com/en-us/azure/iot-suite/iot-suite-remote-monitoring-explore). Currently the simulated Chiller device is fully implemented, the other device methods detailed will be built and validated in the coming months. If you’d like to add or extend the simulated devices yourself you can start with this [tutorial]( https://review.docs.microsoft.com/en-us/azure/iot-suite/iot-suite-remote-monitoring-test) as reference .
```
Comment (timlaverty) [inline README.md]: nice for other devices -> other device methods

Comment (dluc) [inline README.md]: note about style, we usually wrap long lines around the 76th char. It makes the code easier to read in an editor, and easier to comment in a review (i.e. I can add a comment close to the point in question)

File: README.md [hunk 2]
```diff
@@ -19,7 +19,14 @@ Overview
 > Plus, a lot of great functionality will be coming in the next few 
 > months, including:
 >
-> * Extended method support for your simulated devices to help demo and test additional scenarios
+> * Extended method support for your simulated devices to help demo and test additional scenarios. 
+    Details about the Contoso sample devices and rules can be found in the documentation
+    [here](https://docs.microsoft.com/en-us/azure/iot-suite/iot-suite-remote-monitoring-explore).
```
Comment (dluc) [inline README.md]: nit: remove "en-us" from the URL

File: README.md [hunk 3]
```diff
@@ -19,7 +19,14 @@ Overview
...
+    Currently the simulated Chiller device is fully implemented, the other device methods detailed
+    will be built and validated in the coming months. If you’d like to add or extend the simulated
+    devices yourself you can start with this
```
Comment (dluc) [inline README.md]: "," after "yourself"

File: README.md [hunk 4]
```diff
@@ -19,7 +19,14 @@ Overview
...
+    [tutorial]( https://review.docs.microsoft.com/en-us/azure/iot-suite/iot-suite-remote-monitoring-test)
```
Comment (dluc) [inline README.md]: remove "en-us" from URL

Comment (dluc) [reply README.md]: also remove "review" from the URL and the space before "https://"

File: README.md [hunk 5]
```diff
@@ -122,7 +129,7 @@ Common Scenarios
 
 ## All done? Connect a device!
 By default, the solution uses simulated devices. You can start adding your
-own devices with the instructions here: [Connect a physical device](https://docs.microsoft.com/en-us/azure/iot-suite/iot-suite-connecting-devices-linux)
+own devices with the instructions here: [Connect a physical device](https://docs.microsoft.com/azure/iot-suite/iot-suite-connecting-devices-linux)
```
Comment (jillcary) [inline README.md]: The links in the README are listed at the bottom of the markdown file. Please add the link to the bottom of the page with an identifier, for example:
`[build-badge]: https://img.shields.io/travis/Azure/azure-iot-pcs-remote-monitoring-dotnet.svg`

File: README.md [hunk 6]
```diff
@@ -19,7 +19,14 @@ Overview
...
 > months, including:
 >
-> * Extended method support for your simulated devices to help demo and test additional scenarios
+> * Extended method support for your simulated devices to help demo and test additional scenarios. 
+    Details about the Contoso sample devices and rules can be found in the documentation
+    [here](https://docs.microsoft.com/azure/iot-suite/iot-suite-remote-monitoring-explore).
+    Currently the simulated Chiller device is fully implemented, the other device methods detailed
+    will be built and validated in the coming months. If you’d like to add or extend the simulated
+    devices yourself you can start with this
+    [tutorial]( https://review.docs.microsoft.com/azure/iot-suite/iot-suite-remote-monitoring-test)
```
Comment (jillcary) [inline README.md]: Same comment on link style here. (see below)

**Pull request closed without merging**